### PR TITLE
Token names

### PR DESCRIPTION
**Pull request by Abdullah171**
There was an issue in the project that one currency symbol like maxi had mutiple currencies attached to it. So when the user asked the chatbot of what is the price of maxi it used to return only one currency price of that symbol so Now I have added three more custom prompts. If a currency symbol will have two currencies attached to it then the two_tokens prompt will be inserted to the Bot.
Changed the ISO time format to send the request to coin marketcap api also
Added the future price prediction of three more currencies richai etc.
Other thing was client wanted that when he ask the bot about the price of hex and pls currency then it should return the prices upto 5 and 7 decimals so I made that fix using the prompt. 
Also Replaced the gpt-3 model with 4.

### CODE REVIEW

File: app/dependencies/rnn.py [hunk 1]
```diff
@@ -21,10 +21,13 @@ def train_lstm(currency_name):
...
     if historical_json_data is None:
         return
-    
+
     historical_data = get_each_currency_data(historical_json_data, currency_name)
 
-    dataa = historical_data["quotes"]
+    try:
+        dataa = historical_data["quotes"]
+    except None:
```
Comment (Shrhawk) [inline app/dependencies/rnn.py:29]: What is None for ?

File: app/routers/chat.py [hunk 2]
```diff
@@ -52,43 +57,106 @@ async def start_chat(user_input):
...
+            coin_symbol,
+            currency_name,
+            currency_name2,
+            currency_name3,
+            price_coin,
+            price_coin2,
+            price_coin3,
+        )
+
+    elif len(data_list) == 2:
```
Comment (Shrhawk) [inline app/routers/chat.py:125]: please move this to up before 3

File: app/routers/chat.py [hunk 3]
```diff
@@ -52,43 +57,106 @@ async def start_chat(user_input):
...
+                8) If you have the price of pulsechain (PLS) and user asks about the price of PLS then round the price of PLS to 7 decimal places.
+                """,
+            },
+        ]
+
     num_tokens_message = num_tokens_from_string(messages[0]["content"], "cl100k_base")
 
-    if num_tokens_message > 4040:
-        text_splitter = TokenTextSplitter(chunk_size=4040, chunk_overlap=0)
+    if num_tokens_message > 8000:
```
Comment (Shrhawk) [inline app/routers/chat.py:158]: this is correct value ?

File: app/routers/chat.py (hunk 2 above)
Comment (Abdullah171) [reply app/routers/chat.py:125]: ok

File: app/dependencies/rnn.py (hunk 1 above)
Comment (Abdullah171) [reply app/dependencies/rnn.py:29]: sometimes we don't get data from the coinmarket cap api so in that case I don't want to run the lstm model.

File: app/routers/chat.py (hunk 3 above)
Comment (Abdullah171) [reply app/routers/chat.py:158]: gpt-4 has limit of 8192 tokens so I set the limit to 8000

File: app/dependencies/rnn.py (hunk 1 above)
Comment (Shrhawk) [reply app/dependencies/rnn.py:29]: Then catch the general `exception` please and remove `None`

Comment (Abdullah171) [reply app/dependencies/rnn.py:29]: ok

**Pull request merged**